Automattic/sensei
# Add email integrations info

Resolves #6610

## Proposed Changes

This PR adds the MailPoet and AutomateWoo information to the email settings screen.

## Todo

We are still waiting for confirmation for the links (p6rkRX-5HK-p2#comment-5931) but they can be updated in another PR.

## Testing Instructions


1. Go to Sensei LMS -> Settings -> Emails -> Settings
2. Make sure the MailPoet and AutomateWoo info is correct and the links are working.
3. Install and activate MailPoet/AutomateWoo and make sure the links change to "MailPoet/AutomateWoo Settings" and they point to the right screen.

## Pre-Merge Checklist
